agent/config/config.go

```diff
@@ -264,25 +278,26 @@ func EnvironmentConfig() Config {
 	appArmorCapable := utils.ParseBool(os.Getenv("ECS_APPARMOR_CAPABLE"), false)
 
 	return Config{
-		Cluster:                 clusterRef,
-		APIEndpoint:             endpoint,
-		AWSRegion:               awsRegion,
-		DockerEndpoint:          dockerEndpoint,
-		ReservedPorts:           reservedPorts,
-		ReservedPortsUDP:        reservedPortsUDP,
-		DataDir:                 dataDir,
-		Checkpoint:              checkpoint,
-		EngineAuthType:          engineAuthType,
-		EngineAuthData:          NewSensitiveRawMessage([]byte(engineAuthData)),
-		UpdatesEnabled:          updatesEnabled,
-		UpdateDownloadDir:       updateDownloadDir,
-		DisableMetrics:          disableMetrics,
-		DockerGraphPath:         dockerGraphPath,
-		ReservedMemory:          reservedMemory,
-		AvailableLoggingDrivers: availableLoggingDrivers,
-		PrivilegedDisabled:      privilegedDisabled,
-		SELinuxCapable:          seLinuxCapable,
-		AppArmorCapable:         appArmorCapable,
+		Cluster:                 	clusterRef,
+		APIEndpoint:             	endpoint,
+		AWSRegion:               	awsRegion,
+		DockerEndpoint:          	dockerEndpoint,
+		ReservedPorts:           	reservedPorts,
+		ReservedPortsUDP:        	reservedPortsUDP,
+		DataDir:                 	dataDir,
+		Checkpoint:              	checkpoint,
+		EngineAuthType:          	engineAuthType,
+		EngineAuthData:          	NewSensitiveRawMessage([]byte(engineAuthData)),
+		UpdatesEnabled:          	updatesEnabled,
+		UpdateDownloadDir:       	updateDownloadDir,
+		DisableMetrics:          	disableMetrics,
+		DockerGraphPath:         	dockerGraphPath,
+		ReservedMemory:          	reservedMemory,
+		DockerStopTimeoutSeconds: 	(time.Duration(dockerStopTimeoutSeconds)*time.Second),
```

Comment:
This variable name is confusing at the first place. It sounds like a uint and should has the value of seconds. Instead it's a `time.Duration` which is number of microseconds.